.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Label when approved
-        uses: pullreminders/label-when-approved-action@master
+        uses: pullreminders/label-when-approved-action@trunk
```

Comment:
This should be left at `master`, there's no `trunk` branch there.

Reply:
Whoops, reverted in 8d85aa3!